src/node-red.njk

```diff
@@ -97,4 +97,47 @@ sitemapPriority: 0.8
             {% endfor %}
         </div>
     </div>
+</div>
+
+<div class="about w-full ff-bg-light pt-12 pb-24 md:pb-48 md:pt-32">
+     <div class="-mt-28 text-center">
+        <h2>Node-RED</h2>
+    </div>
+    <div class="container m-auto max-w-4xl">
+        <div
+            class="grid grid-cols-1 md:grid-cols-3 pt-12 md:pt-6 gap-x-8" style="align-items: flex-start;">
+            <!-- Node-RED Quick Tips Column -->
+            <div class="roadmap-section consideration pb-6 md:pb-0">
+                <h3 class="centered-headline">Quick Tips</h3>
+                <ul class="list-disc">
+                    <p>
+                        <li><a href="https://flowforge.com/blog/2023/02/3-quick-node-red-tips-1/">Wiring Shortcuts</a>
```

Comment:
Links back into our own website should start not include the domain so for example:

https://flowforge.com/blog/2023/02/3-quick-node-red-tips-1

should be:

/blog/2023/02/3-quick-node-red-tips-1

Otherwise it's easy to end up on the live site when working on a development version which can be confusing.

Otherwise looks good to me.

Reply:
```suggestion
                        <li><a href="/blog/2023/02/3-quick-node-red-tips-1/">Wiring Shortcuts</a>
```